packages/app-mobile/setupQuickActions.ts

```diff
@@ -47,6 +48,7 @@ export default (dispatch: Function, folderId: string) => {
 				noteId: newNote.id,
 				folderId,
 				routeName: 'Note',
+				isNewPhotoNote: data.type === 'New Photo Note',
```

Comment:
```suggestion
				isNewPhotoNote: data.type === 'New photo note',
```

Make sure that the comparison still works with the above changes!

It may also make sense to add an `enum` with the different shortcut item types to he top of the file, then use this enum for comparisons. For example, this `enum` might look similar to the following:
```ts
...

// Example enum that could be used as types for QuickActions 
enum QuickActionTypes {
	NewNote = 'New note',
	NewToDo = 'New to-do',
	...
}

// eslint-disable-next-line @typescript-eslint/ban-types -- Old code before rule was applied
export default (dispatch: Function...
```

Reply:
Sure, I have made the changes now. Please have a look.